### ISSUE TITLE
Error thrown when starting in dev mode

### ISSUE THREAD

**Issue by erezmus** [diversemix/malcolm#17]
Running the start:dev command gives following error on my machine:

```
[nodemon] starting `node dist/main`
[0] [Nest] 4059   - 07/18/2019, 8:34 AM   [NestFactory] Starting Nest application...
[0] [Nest] 4059   - 07/18/2019, 8:34 AM   [InstanceLoader] TypeOrmModule dependencies initialized +116ms
[0] [Nest] 4059   - 07/18/2019, 8:34 AM   [InstanceLoader] AppModule dependencies initialized +2ms
[0] [Nest] 4059   - 07/18/2019, 8:34 AM   [InstanceLoader] GraphQLModule dependencies initialized +2ms
[0] [Nest] 4059   - 07/18/2019, 8:34 AM   [TypeOrmModule] Unable to connect to the database. Retrying (1)... +73ms
[0] /Users/erez/Development/malcolm/server/src/submission/submission.entity.ts:1
[0] (function (exports, require, module, __filename, __dirname) { import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
[0]                                                                      ^
[0] 
[0] SyntaxError: Unexpected token {
[0]     at new Script (vm.js:80:7)
[0]     at createScript (vm.js:274:10)
[0]     at Object.runInThisContext (vm.js:326:10)
[0]     at Module._compile (internal/modules/cjs/loader.js:664:28)
[0]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
[0]     at Module.load (internal/modules/cjs/loader.js:600:32)
[0]     at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
[0]     at Function.Module._load (internal/modules/cjs/loader.js:531:3)
[0]     at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
[1] 
[1] 8:34:32 AM - Found 0 errors. Watching for file changes.
```

**Comment by erezmus** [comment]
if you change the ormconfig.json to:

```
  "entities": ["dist/**/*.entity{.ts,.js}"],
```

then it seems to work

**Comment by erezmus** [comment]
Closed via #34
